src/main/webapp/js/common.js

```diff
@@ -802,3 +802,33 @@ function isBlank(str) {
     }
     return str.trim() === '';
 }
+
+function toggleSingleCollapse(e) {
```

Comment:
Given that this .js file is used by everyone, we should add header comments these three functions.